megamek/unittests/megamek/client/bot/princess/NewtonAerospacePathRankerTest.java

```diff
@@ -58,10 +29,14 @@
 @RunWith(JUnit4.class)
 public class NewtonAerospacePathRankerTest {
 
-    private Princess mockPrincess;
-    private NewtonianAerospacePathRanker mockPathRanker;
-    private IGame mockGame;
-    private GameOptions mockGameOptions;
+    @SuppressWarnings("unused")
+	private Princess mockPrincess;
+    @SuppressWarnings("unused")
+	private NewtonianAerospacePathRanker mockPathRanker;
+    @SuppressWarnings("unused")
+	private IGame mockGame;
+    @SuppressWarnings("unused")
+	private GameOptions mockGameOptions;
```

Comment:
These private variables are part of a 'broken' jUnit test that had it's test method bodies commented out.   My first instinct beyond just deleting the entire broken test was to comment these variables out also to resolve the unused variable warning, however, I felt if keeping the test, it's better to apply the suppress warning to say, "fix me or delete me."  '@SuppressWarning("unused");' is searchable.